src/main/groovy/edu/oregonstate/mist/inventory/InventoryApplication.groovy

```diff
@@ -27,9 +29,13 @@ class InventoryApplication extends Application<InventoryConfiguration> {
         DBIFactory factory = new DBIFactory()
         DBI jdbi = factory.build(environment, configuration.getDataSourceFactory(), "jdbi")
         InventoryDAO inventoryDAO = jdbi.onDemand(InventoryDAO.class)
-        InventoryDAOWrapper inventoryDAOWrapper = new InventoryDAOWrapper(inventoryDAO)
-        environment.jersey().register(new InventoryResource(inventoryDAOWrapper,
-                configuration.api.endpointUri))
+        URI inventorySelfLink = UriBuilder.fromUri(
```

Comment:
I like the idea of having it as a method, but not in the application class. I put it in the configuration class. Let me know what you think :)